queries/metabase/views/014_members_invited_not_validated.sql

```diff
@@ -0,0 +1,67 @@
+-- Utilisateurs invités en tant que membre de structure,
+-- en attente de validation de rattachement,
+-- emails *NON* validés.
+
+-- noqa: disable=LT05
+
+drop table if exists mb_members_invited;
+
+create table mb_members_invited as
```

Comment:
tu as déjà appelé ta table mb_members_invited dans le 011. 
J’ai l’impression que tu pourrais te simplifier la vie en factorisant un peu le truc: 
une premiere table  “mb_members” dont le WHERE serait simplement not mu.is_staff

Ensuite le 011 actuel deviendrait tout simplement un 
`select * from mb_members where mu.is_valid and ss.invited_by_admin`
et celle-ci
`select * from mb_members where not mu.is_valid and ss.invited_by_admin`
du coup, mb_members serait bien une table, mais 011 et 014 pourraient eventuellement être simplement des vues, pour t’éviter de répeter l’indexation

Reply:
j'ai fais le bourrin pour valider le SQL, mais oui évidemment

Reply:
Et il y a eu erreur de ma part dans le nommage de la table : elle doit avoir le même nom que le fichier